src/client/login/reducers/index.ts

```diff
@@ -23,7 +21,6 @@ export const defaultEmailValidator = (email: string) =>
 const initialState: LoginState = {
   email: '',
```

Comment:
Would this be needed still? See also definition at LoginState type.

Reply:
yes. we decoupled keystroke updates to Redux, but the full user email is still saved to Redux when the user clicks "Sign In".

Reply:
The value is referenced when the user attempts OTP verification. It's also good to store the user email in Redux so that other components may reference it if necessary.

Reply:
Okay. Took a while to locate `GET_OTP_EMAIL_SUCCESS`, thanks!